docs/docs/guides/protips.md

```diff
@@ -92,27 +92,19 @@ This allows you to access the remote `8501` port on `localhost:8501` while the C
     </div>
     
     This will forward the remote `8501` port to `localhost:3000`.
+    
+Note, a task's port is forwarded to `localhost` only while the CLI is attached.
 
-[Services](../services.md) provide additional features not offered by tasks,
-such as authorization, load balancing, auto-scaling, an OpenAI-compatible endpoint for models, etc.
-
-Unlike tasks, services are accessible throughout their lifetime, not only when the CLI is attached.
-By default, they are published at `<dstack server URL>/proxy/services/<project name>/<run name>/`.
-Additionally, if your project has a [gateway](../concepts/gateways.md),
-services can be published at a custom domain with HTTPS instead.
-
-So what should you choose for running a web app? Here are some suggestions:
-
-- If you are running a simple app that you only need temporarily, consider **tasks**.
-- If your app needs to be available at all times or if it needs to benefit from advanced features
-  such as authorization or load balancing, use **services**.
-    - If the service will only be accessed by you and other `dstack` users and supports running
-      behind a URL path prefix, **no gateway** is needed.
-    - If the service requires public access, a custom domain, HTTPS, or increased network throughput,
-      **create a gateway** first.
+> Use [tasks](../tasks.md) when you don't need authorization, OpenAI-compatible endpoint,
+custom domain with HTTPS, multiple replicas, and when you don't need to let 
+other users access the endpoint.
+In all other cases, use [services](../services.md).
 
-??? info "Auto-scaling and WebSockets"
-    Services using WebSockets or auto-scaling currently require a gateway.
+!!! info "Gateway"
+    To enable [auto-scaling](reference/dstack.yml/service.md#auto-scaling), or use a custom domain with HTTPS, 
+    set up a [gateway](concepts/gateways.md) before running the service.
+    If you're using [dstack Sky :material-arrow-top-right-thin:{ .external }](https://sky.dstack.ai){:target="_blank"},
+    a gateway is pre-configured for you.
 
```

Comment:
(nit) Considering the text above no longer describes services, mentioning gateways looks irrelevant
```suggestion
```